src/rimage.c

```diff
@@ -88,6 +88,9 @@ int main(int argc, char *argv[])
 		case 'y':
 			image.verify_file = optarg;
 			break;
+		case 'p':
+			image.verify_file = optarg;
```

Comment:
Why is this variable being reused for a different purpose ?

Reply:
thanks, I will refine it.